os/arch/xtensa/src/Makefile

```diff
@@ -69,21 +69,18 @@ ifeq ($(CONFIG_WINDOWS_NATIVE),y)
   ARCH_SRCDIR = $(TOPDIR)\arch\$(CONFIG_ARCH)\src
   TINYARA = "$(TOPDIR)\$(OUTBIN_DIR)\tinyara$(EXEEXT)"
   CFLAGS += -I$(ARCH_SRCDIR)\chip
-  CFLAGS += -I$(ARCH_SRCDIR)\common
```

Comment:
for the code, it looks good to me,  
for commit log, it seems lack of removing unused "common"  path.

Reply:
I agree. But I am sorry I already merged it.